pandaserd/module.py

```diff
@@ -27,7 +34,9 @@ def __init__(self, table, table_name, **kwargs):
         self.align = 'left'
         self.font_color = 'grey60'
         self.bg_color = kwargs.get('bg_color', 'grey')
```

Comment:
The default color `grey` (British spelling) is not part of the `self.html_colors` list, `gray` (US spelling) is.